src/nvim/ui.c

```diff
@@ -70,6 +70,7 @@ bool ui_cb_ext[kUIExtCount];  ///< Internalized UI capabilities.
 static bool has_mouse = false;
 static int pending_has_mouse = -1;
 static bool pending_default_colors = false;
+static bool was_busy = false;
```

Comment:
only `ui_flush` uses this so it should be defined in that scope.

Reply:
yes. I wrote it like this at first. but I was afraid of breaking any rules so I moved it to the top